README.md

```diff
@@ -1,36 +1,108 @@
+processing https://github.com/coala/scripts
+processing https://github.com/coala/documentation
+processing https://github.com/coala/coala-eclipse
+processing https://github.com/coala/meta
+processing https://github.com/coala/school.coala.io
+processing https://github.com/coala/cEPs
+processing https://github.com/coala/coala-decorators-USE-cOALA-UTILS-INSTEAD
+processing https://github.com/coala/coala-emacs
+processing https://github.com/coala/coala-sublime
+processing https://github.com/coala/coala
+processing https://github.com/coala/corobo-answers
+processing https://github.com/coala/asknot-ng
+processing https://github.com/coala/coalaHs
+processing https://github.com/coala/docker-coala-base
+processing https://github.com/coala/gci-leaders
+processing https://github.com/coala/aspect-docs
+processing https://github.com/coala/projects
+processing https://github.com/coala/moonmoon-compose
+processing https://github.com/coala/coala-tutorial
+processing https://github.com/coala/coala-gedit
+processing https://github.com/coala/CoTest
+processing https://github.com/coala/coala-bears
+processing https://github.com/coala/clone-evaluation-helpers
+processing https://github.com/coala/coala-aur-pkgbuild
+processing https://github.com/coala/coala-atom
+processing https://github.com/coala/teams
+processing https://github.com/coala/coala-gui
+processing https://github.com/coala/coAST
+processing https://github.com/coala/community
+processing https://github.com/coala/corobo
+processing https://github.com/coala/coalaCSS
+processing https://github.com/coala/coala-quickstart
+processing https://github.com/coala/rultor-python
+processing https://github.com/coala/coala-ls
+processing https://github.com/coala/landing-frontend
+processing https://github.com/coala/docker-coala-socialx
+processing https://github.com/coala/confer-backend
+processing https://github.com/coala/workshops
+processing https://github.com/coala/bear-docs
+processing https://github.com/coala/coala-html
+processing https://github.com/coala/coala-cmake
 https://github.com/coala/devops: passing
-https://github.com/coala/bear-docs: failing
-https://github.com/coala/docker-coala-base: None
-https://github.com/coala/projects: failing
-https://github.com/coala/coala-html: passing
-https://github.com/coala/coala-ls: failing
-https://github.com/coala/community: passing
-https://github.com/coala/coala-bears: failing
-https://github.com/coala/gh-board: failing
+https://github.com/coala/coala-vs-code: passing
 https://github.com/coala/git-task-list: passing
-https://github.com/coala/coala-eclipse: failing
+https://github.com/coala/gh-board: failing
 https://github.com/coala/PyPrint: passing
-https://github.com/coala/coala: failing
-https://github.com/coala/coala-emacs: failing
 https://github.com/coala/documentation: failing
-https://github.com/coala/corobo: passing
+https://github.com/coala/coala-emacs: failing
+https://github.com/coala/coala: passing
+https://github.com/coala/docker-coala-base: error
```

Comment:
I also get an error for https://github.com/coala/docker-coala-base , but `--verbose` doesnt say what happened.  can it show the problem?